examples/slu/speech_intent_slot/requirements.txt

```diff
@@ -0,0 +1,6 @@
+progress==1.5
```

Comment:
These should go inside the {root}/requirements/requirements_slu.txt and added as tot he setup.py requirements as a subdomain - nemo_toolkit["slu"] = [asr, slu] requirements. Look at TTS for example of this union-like dependency.